Tools/WinMLRunner/src/EventTraceHelper.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <iostream>
+#include <windows.h>
+#include <evntrace.h>
+#include <evntcons.h>
+#include <tchar.h>
+#include <tdh.h>
+#include "ThreadPool.h"
+#include <in6addr.h>
+#include "CommandLineArgs.h"
+
+using namespace std;
+
+class EventTraceHelper
+{
+private:
+    TRACEHANDLE traceHandle;
+    TRACEHANDLE sessionHandle;
+    PEVENT_TRACE_PROPERTIES sessionProperties;
+    ThreadPool threadPool;
+    static DWORD PointerSize;

```

Comment:
>PointerSize [](http://example.com/codeflow?start=17&length=11)

Can this be moved into the cpp file to not pollute the header interface file?

Reply:
Sure